src/autocomplete/EmojiAutocomplete.js

```diff
@@ -29,7 +29,7 @@ export default function EmojiAutocomplete(props: Props): Node {
     [onAutocomplete],
   );
 
-  if (emojiNames.length === 0) {
+  if (filteredEmojis.length === 0) {
```

Comment:
This name is good to fix, because the existing name is inaccurate -- these are emoji objects, not just names.

Currently it's happening in a commit with this commit message:

    emoji [nfc]: Use memoized selector for image emoji.
    
    Introduces a new selector, getActiveImageEmoji which returns a list of
    active image emojis in the correct shape to pass to @zulip/shared.
    
    Also removes the construction of new objects for every matching emoji at
    the return of getFilteredEmojis, as there is now no difference in the
    shape of emoji objects.

That makes it look like this rename has to do with that change -- it makes it sound like the old name was accurate, and the commit is changing what `getFilteredEmojis` returns, from strings to whole emoji objects. Which is confusing for reading the commit, because it turns out not to be true.

Instead, this sort of name change should happen as a separate prep commit.

That also provides a natural space for a commit message to explain that the old name was inaccurate and these are in fact whole emoji objects.

Reply:
I will rename this variable in a separate commit as to not confuse future readers.

Reply:
Split.